aws/resource_aws_cloudwatch_log_stream_test.go

```diff
@@ -65,6 +65,40 @@ func testAccCheckCloudWatchLogStreamDisappears(ls *cloudwatchlogs.LogStream, lgn
 	}
 }
 
+func TestAccAWSCloudWatchLogStream_disappears2(t *testing.T) {
```

Comment:
Nit: We prefer to put the cause for the additional test in the naming instead of a number, e.g.

```suggestion
func TestAccAWSCloudWatchLogStream_disappears_LogGroup(t *testing.T) {
```